portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,6 +65,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int pageSize = getNumComments(request);
     int pageNum = Integer.parseInt(request.getParameter(PAGE_NUM_PARAM));
     String sortBy = request.getParameter(SORT_BY_PARAM);
+    String languageCode = request.getParameter(LANGUAGE_PARAM);
```

Comment:
Perhaps add some server side validation for languageCode? I.e. ensure it's one of the acceptable strings ["en", "zh", "es", etc...], if not default to "en"